### PR TITLE
plotrix community contribution 18

### PR DESCRIPTION
**Pull request by fw98**
Community Contribution for Alex Wan and Junyang Jiang

### CODE REVIEW

File: plotrix.Rmd [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+Alex Wan and Junyang Jiang
+
+```{r echo = FALSE, message=FALSE}
+  library(plotrix)
+```
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
```
Comment (kmsingh-git) [inline plotrix.Rmd]: Please remove this chunk

**Review by kmsingh-git — CHANGES_REQUESTED** [review]
Hey, please remove the setup chunk in the .rmd file, and remove the binary file in your commit

File: plotrix.Rmd (hunk 1 above)
Comment (fw98) [reply plotrix.Rmd]: updated!

Comment (kmsingh-git) [reply plotrix.Rmd]: Wait, your partner has already submitted this in a separate pull request https://github.com/jtr13/cc19/pull/77.
I don't think you need to submit it as well. Right? @bojum

Comment (bojum) [reply plotrix.Rmd]: @kmsingh-git Good catch!! Only one submission needed for a group.

**Comment by bojum** [comment]
@fw98 I merged the branch your partner submitted earlier (see #77 ). I will close this issue. Let me know if there is something you want us to consider in this PR.

File: plotrix.Rmd (hunk 1 above)
Comment (fw98) [reply plotrix.Rmd]: so my partner and i actually did two separate things for our community contribution... this is different than the one he wrote

**Comment by bojum** [comment]
I see. So your partner wrote `GoogleVis_tutorial.Rmd ` and you wrote `plotrix.Rmd`? If this is the case, please remove `GoogleVis tutorial.Rmd` from this PR. Then, @kmsingh-git will check if your branch can be merged into master branch.

**Comment by fw98** [comment]
Okay, I deleted googlevis tutorial.rmd

**Comment by bojum** [comment]
🎉 Merged. Thanks for your contribution!

**Pull request merged**